.github/workflows/publish-helm-chart.yml

```diff
@@ -0,0 +1,42 @@
+name: Publish helm chart
```

Comment:
Maybe one could add a short description that summarize what it means to publish a helm chart here? Writing that the gh-pages branch is used as a Helm chart repository providing an index.yaml file that references `.tgz` files etc.